x/blob/client/testutil/integration_test.go

```diff
@@ -70,8 +122,7 @@ func (s *IntegrationTestSuite) TestSubmitPayForBlob() {
 		{
 			name: "valid transaction",
 			args: []string{
-				hexNamespace,
-				hexBlob,
+				validPropFile.Name(),
```

Comment:
Note: if we add this feature in a non-breaking way, then we can leave this test case untouched so that we have test coverage of the existing single PFB submission via CLI. We can additionally add a new test case that uses multiple PFBs via a file.